velox/core/PlanNode.h

```diff
@@ -598,6 +627,15 @@ class AggregationNode : public PlanNode {
   const std::vector<std::string> aggregateNames_;
   const std::vector<Aggregate> aggregates_;
   const bool ignoreNullKeys_;
+
+  // GroupId column for AggregationNode over a GroupIdNode.
```

Comment:
Perhaps, remove these comments to avoid repetition and a risk of getting out of sync.